common/src/main/java/org/jboss/da/common/util/FileUtils.java

```diff
@@ -0,0 +1,25 @@
+package org.jboss.da.common.util;
```

Comment:
Do we want to depend on this library? I don't like it much because it uses nearly every project and it sometimes happens that there can be compatibility issues. E.g. I solved this issue when 1 library requested Guava 13.0.1 and another Guava 17.x and it was not able to have both of them. So, I use this more defensive approach, when it is easy for me to create method like this.